src/main/java/com/google/devtools/build/lib/analysis/IncompatiblePlatformProvider.java

```diff
@@ -95,6 +105,8 @@ public boolean isImmutable() {
    *
    * <p>This may be null. If it is null, then {@code getTargetsResponsibleForIncompatibility()} is
    * guaranteed to be non-null. It will have at least one element in it if it is not null.
+   *
+   * <p>The list is sorted based on the toString() result of each constraint.
```

Comment:
Probably just reword this to be "stringified label" rather than "toString()".

Reply:
+1

Reply:
Done